base/util_testing.go

```diff
@@ -670,3 +670,33 @@ func ForAllDataStores(t *testing.T, testCallback func(*testing.T, sgbucket.DataS
 		})
 	}
 }
+
+// clusterCompatibility = MajorVersion * 65536 + MinorVersion
+const minCompatVersionForCollections = 7 * 65536
```

Comment:
Might be worth seeing if we can use `decodeClusterVersion()`/`encodeClusterVersion()` instead (in `base/util.go`)